src/main/java/sg/ncl/MainController.java

```diff
@@ -2250,7 +2250,7 @@ public String validateExperiment(
         experimentObject.put("nsFile", "file");
         experimentObject.put("nsFileContent", experimentForm.getNsFileContent());
         experimentObject.put("idleSwap", "240");
-        experimentObject.put("maxDuration", "960");
+        experimentObject.put("maxDuration", experimentForm.getMaxDuration());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "maxDuration" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)